spec/api/v4/imports/imports_spec.rb

```diff
@@ -46,6 +46,13 @@
         let(:import_request) { {"resources" => [{"invalid" => "invalid"}]} }
         run_test!
       end
+
+      response "403", "Forbidden" do
+        let(:HTTP_X_ORGANIZATION_ID) { "wrong_org" }
+        let(:Authorization) { "Bearer #{token.token}" }
+        let(:import_request) { {"resources" => [{"doesn't" => "matter"}]} }
+        run_test!
+      end
```

Comment:
Also not related to the PR, but noticed this edge case was missing in the tests, so I added that.